### PR TITLE
Added ET3 Response form callbacks

### PR DESCRIPTION
**Pull request by JackReevies**
### Change description ###

Added callbacks needed for ET3 response pages

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by hsjhita — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/Et3ResponseHelper.java [hunk 1]
```diff
@@ -23,4 +29,38 @@ private Et3ResponseHelper() {
...
+     * the employment end date if both dates are provided.
+     * @param caseData data for the current case
+     * @return List of validation errors encountered
+     */
+    public static List<String> validateEmploymentDates(CaseData caseData) {
+        ArrayList<String> errors = new ArrayList<>();
+        String startDateStr = caseData.getEt3ResponseEmploymentStartDate();
+        String endDateStr = caseData.getEt3ResponseEmploymentEndDate();
+
+        if (isNullOrEmpty(startDateStr)) {
```
Comment (gauravhmcts) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/Et3ResponseHelper.java:44]: Do we need an error message if start date is empty? Same with end date?

Comment (hsjhita) [reply src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/Et3ResponseHelper.java:44]: Start Date and End date are optional and validation should only take place once values are inputted

Comment (JackReevies) [reply src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/Et3ResponseHelper.java:44]: This is stated in the JavaDoc too

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/Et3ResponseHelper.java [hunk 2]
```diff
@@ -23,4 +29,38 @@ private Et3ResponseHelper() {
...
+            return errors;
+        }
+
+        LocalDate startDate = LocalDate.parse(startDateStr);
+
+        if (startDate.isAfter(LocalDate.now())) {
+            errors.add("Start date must be in the past");
```
Comment (gauravhmcts) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/Et3ResponseHelper.java:51]: Can error messages be constant strings as you defined in tests? They can be defined here and test class can reuse them

**Review by gauravhmcts — APPROVED** [review]

**Pull request merged**